src/Managers/AnimationManager.cpp

```diff
@@ -1131,6 +1131,12 @@ bool CAnimationManager::TestPixels(
 
     auto &frame = direction.m_Frames[frameIndex];
     auto spr = (CSprite *)frame.UserData;
+
+    if (!spr)
+    {
+        return false;
+    }
+

```

Comment:
Take a look at the line 2119. Copy and paste that instead.

Reply:
i have done the change